<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QWidget" name="widget" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnPalette">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>120</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image.jpg</normaloff>
          <normalon>:/image.jpg</normalon>
          <disabledoff>:/image.jpg</disabledoff>
          <disabledon>:/image.jpg</disabledon>
          <activeoff>:/image.jpg</activeoff>
          <activeon>:/image.jpg</activeon>
          <selectedoff>:/image.jpg</selectedoff>
          <selectedon>:/image.jpg</selectedon>image.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>130</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelError">
        <property name="text">
         <string>Округление</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gBoxRGB">
      <property name="title">
       <string>RGB</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <widget class="QSpinBox" name="sBoxRGBb">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sliderRGBr">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="sliderRGBb">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sliderRGBg">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="sBoxRGBg">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="sBoxRGBr">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelRGBb">
           <property name="text">
            <string>B</string>
           </property>
           <property name="buddy">
            <cstring>sliderRGBb</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRGBg">
           <property name="text">
            <string>G</string>
           </property>
           <property name="buddy">
            <cstring>sliderRGBg</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelRGBr">
           <property name="text">
            <string>R</string>
           </property>
           <property name="buddy">
            <cstring>sliderRGBr</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gBoxHSV">
      <property name="title">
       <string>HSV</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QSlider" name="sliderHSVv">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="sBoxHSVs">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sliderHSVs">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sliderHSVh">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>359</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="sBoxHSVv">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="sBoxHSVh">
           <property name="maximum">
            <number>359</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelHSVh">
           <property name="text">
            <string>H</string>
           </property>
           <property name="buddy">
            <cstring>sliderHSVh</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelHSVs">
           <property name="text">
            <string>S</string>
           </property>
           <property name="buddy">
            <cstring>sliderHSVs</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelHSVv">
           <property name="text">
            <string>V</string>
           </property>
           <property name="buddy">
            <cstring>sliderHSVv</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gBoxLUV">
      <property name="title">
       <string>Lu'v'</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="1">
          <widget class="QSlider" name="sliderLUVu">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="sliderLUVv">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="sBoxLUVl">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sliderLUVl">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="sBoxLUVv">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="sBoxLUVu">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelLUVl">
           <property name="text">
            <string>L</string>
           </property>
           <property name="buddy">
            <cstring>sliderLUVl</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLUVu">
           <property name="text">
            <string>u'</string>
           </property>
           <property name="buddy">
            <cstring>sliderLUVu</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelLUVv">
           <property name="text">
            <string>v'</string>
           </property>
           <property name="buddy">
            <cstring>sliderLUVv</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gBoxCMYK">
      <property name="title">
       <string>CMYK</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QSpinBox" name="sBoxCMYKm">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="sBoxCMYKy">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="sBoxCMYKk">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="sBoxCMYKc">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sliderCMYKm">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="sliderCMYKy">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="sliderCMYKk">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sliderCMYKc">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelCMYKc">
           <property name="text">
            <string>C</string>
           </property>
           <property name="buddy">
            <cstring>sliderCMYKc</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCMYKm">
           <property name="text">
            <string>M</string>
           </property>
           <property name="buddy">
            <cstring>sliderCMYKm</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelCMYKy">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="buddy">
            <cstring>sliderCMYKy</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelCMYKk">
           <property name="text">
            <string>K</string>
           </property>
           <property name="buddy">
            <cstring>sliderCMYKk</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRGBr</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxRGBr</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRGBg</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxRGBg</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRGBb</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxRGBb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHSVh</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxHSVh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCMYKc</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxCMYKc</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCMYKm</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxCMYKm</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCMYKy</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxCMYKy</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>676</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCMYKk</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxCMYKk</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>705</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLUVl</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxLUVl</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLUVu</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxLUVu</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLUVv</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sBoxLUVv</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
